backend/amd64/emit.ml

```diff
@@ -2078,9 +2246,23 @@ let emit_item : Cmm.data_item -> unit = function
   | Csingle f -> D.float32 f
   | Cdouble f -> D.float64 f
   (* SIMD vectors respect little-endian byte order *)
-  | Cvec128 { high; low } ->
-    D.float64_from_bits low;
-    D.float64_from_bits high
+  | Cvec128 { word0; word1 } ->
```

Comment:
(Again, I am not sure it is the
natural order for the fields, but
obviously does not matter / can
be ignored.)